cloud_storage_handler/utils.py

```diff
@@ -0,0 +1,37 @@
+"""Utility functions for the Cloud Storage Handler package."""
+
+import os
+from pathlib import Path
+
+from foca import Foca
+
+from cloud_storage_handler.custom_config import CustomConfig
+from cloud_storage_handler.exceptions import ConfigNotFoundError
+
+
+def get_config_path() -> Path:
+    """Get the configuration path.
+
+    Returns:
+      The path of the config file.
+    """
+    # Determine the configuration path
+    if config_path_env := os.getenv("CSH_FOCA_CONFIG_PATH"):
+        return Path(config_path_env).resolve()
+    else:
+        return (Path(__file__).parents[1] / "deployment" / "config.yaml").resolve()
+
+
+def get_custom_config() -> CustomConfig:
+    """Get the custom configuration.
+
+    Returns:
+      The custom configuration.
+    """
+    conf = Foca(config_file=get_config_path()).conf
+    try:
+        return CustomConfig(**conf.custom)
+    except AttributeError:
+        raise ConfigNotFoundError(
+            "Custom configuration not found in config file."
+        ) from None
```

Comment:
Possibly/probably not needed.

In any case, there is no need to create a new FOCA instance. Either access the config from the app instance that is already created, in `cloud_storage_handler.main.main()` or via `current_app` when in app context in a controller, or by manually pushing app context when outside of app context.